### DISCUSSION TITLE
INI file in OMNeT++ 6.0.3

### DISCUSSION THREAD

**Discussion by pjpf** [omnetpp/omnetpp#1165] [Q&A] [answered]
Hello,
I am using OMNeT++ 6.0.3 and it I have the following error:

Starting...
[...]
<!> Error: Syntax error in section heading, 'omnetpp.ini' line 1
[...]
End.
Simulation terminated with exit code: 1

Lien 1 in omnetpp.ini is: 
[General] 
network = MobilityShowcase
and it is not marked as an error !

By the way, I have a network which is called MobilityShowcase!

Please, help me !

Best regards.

--Paulo--

**Comment by rhornig** [comment]
Are you 100% sure that OMNeT++ is reading the INI file you think it is reading?

**Reply by pjpf** [reply]
yes, sure !

**Reply by rhornig** [reply, marked as answer]
Gee. this is a bug. see #1167 . For a workaround, remove the space at the end of line in `[General]`

**Reply by rhornig** [reply]
fixed in 6.1 branch
